src/util/projects.tsx

```diff
@@ -21,3 +23,14 @@ export const getSubpageFromUrl = (url: string) => {
   }
   return undefined;
 };
+
+export const isProjectEmpty = (project: LxdProject) => {
+  if (!project.used_by) {
+    return true;
+  }
+
+  return (
+    project.used_by.filter((item) => !item.includes("/1.0/profiles/"))
```

Comment:
[A project with profiles](https://lxd-ui-279.demos.haus/ui/baz/configuration) is non-empty too, and as such, can't be deleted:

![image](https://user-images.githubusercontent.com/56583786/230426896-dc880e76-cfd3-4990-8536-09123b47d081.png)

Reply:
So we should only exclude the default profile here, right.